### ISSUE TITLE
Gene Expression Resources Page -- Text needs to be Black

### ISSUE THREAD

**Issue by maxglycine** [soybase/jekyll-soybase#163]
The Gene Expression Resources for Soybean page (https://dev.soybase.org/tools/expression/) text needs to have the font color changed for contrast reasons.  The text should be Black, not Grey, for section 508 compliance.  I know I sent this out by email, but I figured that there should be an issue for it so it did not get forgotten.  It looks like the font-color was intentionally overridden by a local CSS section.  So just dropping the font-color in the CSS section may do it.

@StevenCannon-USDA @jd-campbell @ChenProm-USDA

**Comment by ChenProm-USDA** [comment]
I'm currently working on this issue (branch expression-page-contrast).  I'm adding the navigation text color and keyboard navigation issue on this branch as well.  Most of it is complete just trying to solve some responsiveness issues.
